test/fixtures/game_teams_truncated.csv

```diff
@@ -0,0 +1,300 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
```

Comment:
I thought we need all three data files

Reply:
The teams file is pretty short already (33 lines)

Reply:
Could we make these shorter? So it's easier to manually calculate things like percentage_home_wins for our testing.